contexts/credentials/v2

```diff
@@ -92,6 +92,28 @@
       }
     },
 
+    "JsonSchema2023": {
+      "@id":
+        "https://www.w3.org/ns/credentials#JsonSchema2023",
+      "@context": {
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type"
+      }
+    },
+
+    "CredentialSchema2023": {
+      "@id":
+        "https://www.w3.org/ns/credentials#CredentialSchema2023",
```

Comment:
```suggestion
    "VerifiableCredentialSchema2023": {
      "@id":
        "https://w3.org/2018/credentials#VerifiableCredentialSchema2023",
```

The WG has decided to not define `Credential`, and so the suggestion here is to align w/ where the WG is currently. Also, same commentary as "vocabulary split" issue in https://github.com/w3c/vc-data-model/pull/1178/files#r1257273393.